schemas/common/dataImportEventTypes.json

```diff
@@ -27,6 +28,7 @@
     "DI_INVENTORY_INSTANCE_UPDATED",
     "DI_INVENTORY_INSTANCE_CREATED_READY_FOR_POST_PROCESSING",
     "DI_INVENTORY_HOLDINGS_CREATED_READY_FOR_POST_PROCESSING",
+    "DI_INVENTORY_AUTHORITY_CREATED_READY_FOR_POST_PROCESSING",
```

Comment:
Also I'm not sure about this event. Will we do some postprocessing for authority records?